contracts/rootchain/RootChain.sol

```diff
@@ -76,19 +75,18 @@ contract RootChain is Ownable {
 
     constructor() public
     {
-        currentChildBlock = 1;
+        lastCommittedBlock = 0;
         depositNonce = 1;
-        lastParentBlock = block.number;
-
         minExitBond = 10000;
     }
 
     // @param blocks 32 byte merkle roots
-    function submitBlock(bytes blocks)
+    // @param blockNum the block number of the first header
+    function submitBlock(bytes blocks, uint256 blockNum)
```

Comment:
Also do we still want/need a insertBlock function that can be called by anyone with validator signature?

Reply:
I have been thinking about this. I'll add it back in. The only case where i would see it get called is if the operator was to go offline but has already minted blocks. 

Gossiping invalid signatures is the same as submitting an invalid block so i think it's okay